content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -808,6 +808,40 @@ Example: `node.cloudprovider.kubernetes.io/shutdown: "NoSchedule"`
 
 If a Node is in a cloud provider specified shutdown state, the Node gets tainted accordingly with `node.cloudprovider.kubernetes.io/shutdown` and the taint effect of `NoSchedule`.
 
+### nfd.node.kubernetes.io/master.version
+
+Example: `nfd.node.kubernetes.io/master.version: v0.6.0`
+
+This annotation defines version of the NFD-Master instance running on the node.
+It is used for informative use only.
+
+### nfd.node.kubernetes.io/worker.version
+
+Example: `nfd.node.kubernetes.io/worker.version: v0.4.0`
+
+This annotation defines version of the NFD-Worker instance running on the node.
+It's used for informative use only.
+
+### nfd.node.kubernetes.io/feature-labels
+
+Example: `nfd.node.kubernetes.io/feature-labels: cpu-cpuid.ADX,cpu-cpuid.AESNI,cpu-hardware_multithreading,kernel-version.full,system-os_release.ID ...`
+
+This annotation defines comma-separated list of node labels managed by NFD.
+NFD uses this internally so it must not be edited by users.
+
+### nfd.node.kubernetes.io/extended-resources
+
+Example: `nfd.node.kubernetes.io/extended-resources: sgx.intel.com/epc`
+
+This annotation defines a comma-separated list of node extended resources managed by NFD.
+NFD uses this internally so it must not be edited by users.
+
+{{< note >}}
```

Comment:
Adding this at the beginning makes more sense, IMO, since we are referencing nfd multiple times.

Reply:
I have mentioned it in beginning earlier as a [preview](https://github.com/kubernetes/website/issues/31612#issuecomment-1491997100) but later, as per the doc all notes are after the statements ,I added it in end.

I agree it will give more info.

Reply:
@divya-mohan0209 any other information that i can add in note ? or is just good enough as it is?